src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanUpFixer.cs

```diff
@@ -13,6 +14,13 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup
     /// </summary>
     internal abstract class CodeCleanUpFixer : ICodeCleanUpFixer
     {
-        public abstract Task<bool> FixAsync(ICodeCleanUpScope scope, ICodeCleanUpExecutionContext context, CancellationToken cancellationToken);
+        [Obsolete("use one without cancellationtoken", error: true)]
+        public virtual Task<bool> FixAsync(ICodeCleanUpScope scope, ICodeCleanUpExecutionContext context, CancellationToken _)
+        {
+            // cancellation token will be removed in next API update
```

Comment:
Tracking bug or something to link to on the editor side?

Reply:
sure. I can create one.

Reply:
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/841276